docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -14,6 +14,7 @@
    openvino_docs_MO_DG_prepare_model_convert_model_Cutting_Model
    openvino_docs_MO_DG_Additional_Optimization_Use_Cases
    openvino_docs_MO_DG_FP16_Compression
+   openvino_docs_MO_DG_prepare_model_MO_Python_API
```

Comment:
I would suggest to have `openvino_docs_MO_DG_MO_Python_API` or `openvino_docs_MO_DG_Python_API`